src/EnergyPlus/OutputProcessor.cc

```diff
@@ -1672,6 +1672,14 @@ namespace OutputProcessor {
 		cCurrentModuleObject = "Meter:Custom";
 		NumCustomMeters = GetNumObjectsFound( cCurrentModuleObject );
 
+        //make list of names for all Meter:Custom since they cannot refer to other Meter:Custom's
+		std::vector<std::string> namesOfMeterCustom;
```

Comment:
@shorowit thanks for letting me know. That does change my perspective on the value of optimizing this.